src/azul/indexer/index_service.py

```diff
@@ -477,6 +493,33 @@ def aggregate(self, tallies: CataloguedTallies):
                 break
         writer.raise_on_errors()
 
+    def replicate(self,
+                  catalog: CatalogName,
+                  replicas: list[Replica],
```

Comment:
```suggestion
                  replicas: list[Replica]
```

and this fits on one line. Look for other cases of unnecessarily wrapped method signatures.

Reply:
It no longer fits on one line on my `fixup`, but I did find another method to unwrap :+1: